test.py

```diff
@@ -200,6 +200,10 @@ def parse_cmd_line() -> argparse.Namespace:
                         help='Let me manually run the test executable at the moment this script would run it')
     parser.add_argument('--byte-limit', action="store", default=randint(0, 2000), type=int,
                         help="Specific byte limit for failure injection (random by default)")
+    parser.add_argument("--pytest-arg", action='store', type=str,
+                        default=None, dest="pytest_arg",
+                        help="Additional command line arguments to pass to "
+                        "pytest, for example ./test.py --pytest-arg=\"-v -x\"")
```

Comment:
Can be sent as an independent enhancement so to expedite it to master

Reply:
Sent as a separate PR.